react-ui/src/components/Layouts/InputWorkerTimes/WorkerInputpage.css

```diff
@@ -0,0 +1,30 @@
+
+/* .SelectSchools{
```

Comment:
Hi Kyle,
When code is commented out, it's usually good to leave a description of why it's still kept (its purpose) OR to just remove it to prevent unnecessary growth in file sizes.

Reply:
Noted for future iterations of coding